foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -67,6 +78,7 @@ class AddOnProgress(BaseNotification):
     "addon-install-blocked-notification": AddOnInstallBlocked,
     "addon-install-confirmation-notification": AddOnInstallConfirmation,
     "addon-install-complete-notification": AddOnInstallComplete,
+    "appMenu-addon-installed-notification": AddOnInstallComplete,
```

Comment:
We should add a comment to indicate that we can clean up anything that not needed once ESR version is later than 63